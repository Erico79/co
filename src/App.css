.App .jumbotron {
  border-radius: 0px;
  background-color: #fff;
}

.App .jumbotron > img {
  width: 100%;
}