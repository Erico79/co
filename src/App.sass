@import './styles/variables.sass'

.wrapper
    margin-top: 80px