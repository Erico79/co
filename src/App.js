import React, { Component } from "react";
import { Switch, Route } from "react-router-dom";

import { Container } from "reactstrap";

import Header from "./components/ui/Header/Header";
import Home from "./components/pages/Home/Home";
import MerryGoRound from "./components/pages/MerryGoRound/MerryGoRound";
import Registration from "./components/pages/Registration/Registration";
import "./App.sass";

class App extends Component {
  render() {
    return (
      <div className="App">
        <Header />

        <Container className="wrapper">
          <Switch>
            <Route exact path="/register" component={Registration} />
            <Route exact path="/merry-go-round" component={MerryGoRound} />
            <Route exact path="/" component={Home} />
          </Switch>
        </Container>
      </div>
    );
  }
}

export default App;
