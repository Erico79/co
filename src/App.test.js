import React from "react";
import App from "./App";
import { shallow } from "enzyme";

it("should render correctly", () => {
  const component = shallow(<App />);
  expect(component).toMatchSnapshot();
});
