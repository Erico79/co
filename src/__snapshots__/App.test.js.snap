// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="App"
>
  <withRouter(Header) />
  <Container
    className="wrapper"
    tag="div"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/register"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/merry-go-round"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
    </Switch>
  </Container>
</div>
`;
